lib/segments/pid.rb

```diff
@@ -12,8 +12,8 @@ class HL7::Message::Segment::PID < HL7::Message::Segment
     convert_to_ts(value)
   end
   add_field :admin_sex do |sex|
-    unless /^[FMOUANC]$/.match(sex) || sex == nil || sex == ""
-      raise HL7::InvalidDataError.new( "bad administrative sex value (not F|M|O|U|A|N|C)" )
+    unless /^[FMOUANCX]$/.match(sex) || sex == nil || sex == ""
```

Comment:
can you also update the [specs](https://github.com/PrimaryDotHealth/ruby-hl7/blob/master/spec/pid_segment_spec.rb#L13) please?